### PR TITLE
fix stow stroage config in copiliot container

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Fixed copiliot stow config flags, If stow config is passed then we don't need to set other flags

Screenshot 2022-03-18 at 11 58 02 AM

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/2239

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: go/tasks/pluginmachinery/flytek8s/copilot.go [hunk 1]
```diff
@@ -63,23 +63,24 @@ func CopilotCommandArgs(storageConfig *storage.Config) []string {
...
+	if len(storageConfig.InitContainer) > 0 {
+		commands = append(commands, fmt.Sprintf("--storage.container=%s", storageConfig.InitContainer))
+
+	}
+	commands = append(commands, fmt.Sprintf("--storage.type=%s", storageConfig.Type))
+
+	if storageConfig.Type == storage.TypeStow {
+		var cmd = ""
+		for key, val := range storageConfig.Stow.Config {
+			cmd += fmt.Sprintf("%s='%s',",key,val)
```
Comment (EngHabu) [inline go/tasks/pluginmachinery/flytek8s/copilot.go:78]: avoid the extra ',' at the end... don't think this will work

File: go/tasks/pluginmachinery/flytek8s/copilot.go [hunk 2]
```diff
@@ -63,23 +63,24 @@ func CopilotCommandArgs(storageConfig *storage.Config) []string {
...
 		commands = append(commands, "--storage.enable-multicontainer")
 	}
+	if len(storageConfig.InitContainer) > 0 {
+		commands = append(commands, fmt.Sprintf("--storage.container=%s", storageConfig.InitContainer))
+
+	}
+	commands = append(commands, fmt.Sprintf("--storage.type=%s", storageConfig.Type))
+
+	if storageConfig.Type == storage.TypeStow {
+		var cmd = ""
```
Comment (EngHabu) [inline go/tasks/pluginmachinery/flytek8s/copilot.go:76]: You should keep the check for `if len(storageConfig.Stow.Config) > 0 && len(storageConfig.Stow.Kind) > 0 {` here as well because there is legacy way and new way of configuring stow... and having a Stow.Config is the "new" way...

**Review by ckiosidis — DISMISSED** [review]
LGTM, there also seems to be an error on the build

**Review by ckiosidis — APPROVED** [review]

**Review by EngHabu — APPROVED** [review]
